functions/initialiseEasyGame.js

```diff
@@ -0,0 +1,27 @@
+/** initialises the hard mode **/
+/*
+* this function:
+*   1. overrides the display property of the .extraCardContainer from 'none' to 'block'
+*   2. override the number of cards that the game will be using from 16 to 32
+*   2. enlarges the cardImageArray to 32 cards (16 pairs)
+*   3. shuffle the array
+*
+ */
+function initialiseEasyGame (){
```

Comment:
Adjust whitespace

Functions should be like this

function name() {

}

So no space after function name, space after params